src/common/request.ts

```diff
@@ -1,91 +1,38 @@
 "use strict";
 
-import { parse, Url } from "url";
-import { logger } from "./logger";
+import * as Popsicle from "popsicle/dist/common";
```

Comment:
It's funny, because it compiles fine on my computer, but travis keeps failing because of some of the dependencies in popsicle.  Need to investigate further.